src/librustc_codegen_utils/symbol_names.rs

```diff
@@ -613,6 +613,9 @@ impl fmt::Write for SymbolPrinter<'_, '_> {
                 // for ':' and '-'
                 '-' | ':' => self.path.temp_buf.push('.'),
 
+                // Avoid segmentation fault on some platforms, see #60925.
+                'm' if self.path.temp_buf.ends_with(".llv") => self.path.temp_buf.push_str("$6d$"),
```

Comment:
Ooops, turns out this should be `$u6d$` (i.e. note the `u`).

Reply:
I've submitted #61423 that changes this.